bin/rename-project

```diff
@@ -43,12 +43,13 @@ while true; do
 done
 
 # -----------------------------------------------------------------------------
-# The core of the script which renames a few things.
+# The core of the script.  Rename a few things, delete stale `/tmp` contents.
```

Comment:
I think we can drop the `. Rename a few things, delete stale `/tmp` contents.` part.